src/analysis/network/qgslinevectorlayerdirector.h

```diff
@@ -37,20 +32,35 @@ class ANALYSIS_EXPORT QgsLineVectorLayerDirector : public QgsGraphDirector
 
   public:
 
+    /** Road direction
+     * Road can be one-way with direct flow (one can move only from the start
+     * point to the end point), one-way with reversed flow (one can move only
+     * from the end point to the start point) and bidirectional or two-way
+     * (one can move in any direction)
+     */
+    enum RoadDirection
```

Comment:
Very nice! Can I suggest DirectionForward, DirectionReversed and DirectionBoth would be simpler names to understand?